src/bull/bull.service.ts

```diff
@@ -7,6 +7,8 @@ import { PrismaService } from '../prisma/prisma.service';
 import { BullQueueService } from './bull-queue.service';
 import { Logger } from '@nestjs/common';
 
+const MaximumDelaySec = 5;
```

Comment:
A commonly accepted standard practice is to uppercase constants (see https://github.com/airbnb/javascript#naming--uppercase)

```suggestion
const MAXIMUM_DELAY_IN_SECONDS = 5;
```